src/main/java/org/example/onlinebookstore/controller/BookController.java

```diff
@@ -14,42 +19,58 @@
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Online book store", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(summary = "Create a new book",
+            description = "Inserting the book into DB")
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
-    public BookDto createBook(@RequestBody CreateRequestBookDto bookDto) {
+    public ResponseBookDto createBook(@RequestBody @Valid RequestCreateBookDto bookDto) {
         return bookService.save(bookDto);
     }
 
+    @Operation(summary = "Get all books",
+            description = "Displays all sorted available books")
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<ResponseBookDto> getAll(@RequestParam(defaultValue = "0") int page,
+                                @RequestParam(defaultValue = "20") int size,
+                                @RequestParam(defaultValue = "title") String field) {
+        return bookService.findAll(PageRequest.of(page, size, Sort.by(field).ascending()));
```

Comment:
```suggestion
    public Page<ResponseBookDto> getAll(Pageble pageble) {
        return bookService.findAll(pageble);
```
you can replace it using pageable